sdk/storage/Azure.Storage.DataMovement/tests/DownloadChunkHandlerTests.cs

```diff
@@ -3,15 +3,9 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
 using Moq;
 using NUnit.Framework;
-using Azure.Storage.DataMovement;
-using Azure.Core.Pipeline;
-using System.Diagnostics;
-using Azure.Storage.Test;
```

Comment:
Comment in randome place: I think maybe we should add a transfer level test for this scenario as well? Something like my `StartTransfer_AwaitCompletion_Failed` but setup in a way to force multiple chunks? This test is how we originally caught this bug but then it was changed to properly call the single download path so we lost the coverage of the chunk download path.

Reply:
Yeah I can add that